### PR TITLE
7961 responsive movie banner

### LINKED ISSUE #7961: Movie banner on mobile

### PR DESCRIPTION
**Pull request by RemcoSimonides**
Fix #7961

### CODE REVIEW

File: apps/festival/festival/src/app/marketplace/title/view/view.component.html [hunk 1]
```diff
@@ -8,27 +8,27 @@
             <org-chip [org]="org"></org-chip>
           </ng-container>
         </section>
-        <section fxLayout fxLayout.lt-sm="column">
+        <section fxLayout="row wrap" fxLayoutAlign.xs="center">
           <ng-container *ngIf="(eventId$ | async) as eventId; else request">
-            <button mat-button  class="translucent-button" [disabled]="requestSent" (click)="requestAskingPrice(movie.id)">
+            <button mat-button class="translucent" [disabled]="requestSent" (click)="requestAskingPrice(movie.id)">
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/marketplace/title/view/view.component.html:13]: I didn't commit the changes properly in my previous PR to change `translucent-button` to `translucent`.

File: apps/festival/festival/src/app/marketplace/title/view/view.component.html [hunk 2]
```diff
@@ -8,27 +8,27 @@
...
             <wishlist-button small [movieId]="movie.id"></wishlist-button>
             <a ongoing-event-button [routerLink]="['/event', eventId, 'r', 'i']">Ongoing Screening</a>
           </ng-container>
           <ng-template #request>
             <event-request-screening [movieId]="movie.id" animated></event-request-screening>
-            <button mat-button class="translucent-button" [disabled]="requestSent" (click)="requestAskingPrice(movie.id)">
+            <button mat-button class="translucent" [disabled]="requestSent" (click)="requestAskingPrice(movie.id)">
               <mat-icon svgIcon="local_offer"></mat-icon>
-              <span class="linear-wipe">Request Asking Price</span>
+              <span *ngIf="gtSm$ | async" class="linear-wipe">Request Asking Price</span>
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/title/view/view.component.html:24]: why not using `fxHide.gtsm` ?

File: apps/festival/festival/src/app/marketplace/title/view/view.component.scss [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+
 movie-header-cta {
   width: 100%;
   margin-bottom: 16px;
 
   .orgs {
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+    @include layout-bp(xs) {
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/title/view/view.component.scss:11]: I prefer to put the media queries at the root of the css file

File: apps/festival/festival/src/app/marketplace/title/view/view.component.ts [hunk 4]
```diff
@@ -61,14 +62,17 @@ export class MarketplaceMovieViewComponent  {
...
   constructor(
     private route: ActivatedRoute,
     private movieService: MovieService,
     private orgService: OrganizationService,
     private eventService: EventService,
     private dialog: MatDialog,
     private cdr: ChangeDetectorRef,
-    private analytics: AnalyticsService
+    private analytics: AnalyticsService,
+    private breakpointsService: BreakpointsService
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/title/view/view.component.ts:75]: I don't think you need it

File: libs/movie/src/lib/movie/components/header/header.component.scss [hunk 5]
```diff
@@ -8,8 +10,16 @@ header {
     background-image: linear-gradient(to right, black, transparent);
 
     .movie-info{
-      .poster {
-        height: 400px;
+      @include layout-bp(gt-sm) {
+        .poster {
+          height: 400px;
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/header/header.component.scss:15]: It's usually better to use properties like `min()`, `max()` or `clamp()` for sizing than media queries with fixed value

File: libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html [hunk 6]
```diff
@@ -1,40 +1,68 @@
-<section fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="24px">
-  <ng-container *ngIf="(links | promotionalLinks: movie | async); let _links">
-    <ng-container *ngIf="videos; else noVideosBtn">
-      <button mat-button (click)="scrollToFooter()">
+<ng-container *ngIf="(links | promotionalLinks: movie | async); let _links">
+
+  <ng-container *ngIf="xs$ | async; else gtxs">
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html:3]: This should be managed by CSS. HTML is not CSS. Use the right tool for the right task.
You can use properties like `display: none` to remove element from the layout.

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]

File: libs/movie/src/lib/movie/components/header/header.component.scss (hunk 5 above)
Comment (RemcoSimonides) [reply libs/movie/src/lib/movie/components/header/header.component.scss:15]: The layout changes from row to column. I'm not able to make these functions work with that structure change.
image

image

File: libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html (hunk 6 above)
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html:3]: Either separate component or using `display: none` as I mention earlier

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**